Formula/borgbackup.rb

```diff
@@ -25,14 +24,14 @@ class Borgbackup < Formula
   depends_on "xxhash"
   depends_on "zstd"
 
-  resource "packaging" do
-    url "https://files.pythonhosted.org/packages/df/86/aef78bab3afd461faecf9955a6501c4999933a48394e90f03cd512aad844/packaging-21.0.tar.gz"
-    sha256 "7dc96269f53a4ccec5c0670940a4281106dd0bb343f47b7471f779df49c2fbe7"
+  resource "msgpack" do
+    url "https://files.pythonhosted.org/packages/61/3c/2206f39880d38ca7ad8ac1b28d2d5ca81632d163b2d68ef90e46409ca057/msgpack-1.0.3.tar.gz"
+    sha256 "51fdc7fb93615286428ee7758cecc2f374d5ff363bdd884c7ea622a7a327a81e"
   end
 
-  resource "pyparsing" do
-    url "https://files.pythonhosted.org/packages/c1/47/dfc9c342c9842bbe0036c7f763d2d6686bcf5eb1808ba3e170afdb282210/pyparsing-2.4.7.tar.gz"
-    sha256 "c203ec8783bf771a155b207279b9bccb8dea02d8f0c9e5f8ead507bc3246ecc1"
```

Comment:
hmm, not relevant any more for this PR (but maybe for an upcoming 1.1.18 PR): why was it `pyparsing` here, shouldn't it have been `packaging`?

Reply:
I don't think Homebrew publishes old stable versions in this repo. Not sure why pyparsing was there before.

Reply:
Homebrew's `bump-formula-pr` uses `pipgrip` to detect dependency tree for PyPI formulae.

This may be a dependency of `borgbackup -> packaging -> pyparsing`.
https://github.com/pypa/packaging/blob/21.3/setup.py#L51
```py
    install_requires=["pyparsing>=2.0.2,!=3.0.5"],  # 2.0.2 + needed to avoid issue #91
```

Reply:
So you guys want `pyparsing` added back? When testing locally it wasn't needed, but I'm happy to add it back if there is a good reason. If we aren't sure I'd only add it back if a concrete problem should come up.

Reply:
guess they want all the dependencies listed, so it would be `packaging` **and** `pyparsing` (indirect via packaging).

Reply:
No problem. Added this back. Hope we can get this PR out now.